otter.install4j

```diff
@@ -663,6 +665,32 @@ return console.askYesNo(message, true);
                 </serializedBean>
                 <condition />
               </action>
+              <action name="" id="257" customizedId="" beanClass="com.install4j.runtime.beans.actions.files.DeleteFileAction" enabled="true" commentSet="false" comment="" actionElevationType="elevated" rollbackBarrier="false" rollbackBarrierExitCode="0" multiExec="false" failureStrategy="1" errorMessage="">
+                <serializedBean>
+                  <java class="java.beans.XMLDecoder">
+                    <object class="com.install4j.runtime.beans.actions.files.DeleteFileAction">
+                      <void property="files">
+                        <array class="java.io.File" length="2">
+                          <void index="0">
+                            <object class="java.io.File">
+                              <string>${installer:sys.appdataDir}${installer:sys.fileSeparator}ProjectOtter</string>
+                            </object>
+                          </void>
+                          <void index="1">
+                            <object class="java.io.File">
+                              <string>${installer:sys.userHome}/.config/ProjectOtter</string>
```

Comment:
You're using `/`s here and fileSeparator five lines up.  Intentional?

Reply:
Yes. the /.config one is for linux the one with fileSeparator is for Windows and MacOS.